### DISCUSSION TITLE
Integrates the Trix Editor with LiveWire

### DISCUSSION THREAD

**Discussion by midomario** [livewire/livewire#5429] [1. Help] [answered]
Hi, when I use any Rich Text Editor (wysiwyg) inside a form that uses livewire, If I click inside the textarea everything is good but if I click or type inside another input at the same form the  Editor (wysiwyg) tools disapears ?

**Comment by joshhanley** [comment, marked as answer]
@midomario yeah Livewire and any Javascript libraries that modify the DOM conflict. So you have to make use of `wire:ignore` to ensure Livewire doesn't touch it. See here for details https://laravel-livewire.com/docs/2.x/alpine-js#ignoring-dom-changes

If you have access to the screencasts, there is one on getting Trix running with Livewire https://laravel-livewire.com/screencasts/s4-rich-text

Hope this helps!

**Reply by midomario** [reply]
Thank you so much sir.

**Reply by joshhanley** [reply]
@midomario no worries! 🙂
